modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -51,6 +52,20 @@ The new open source plugin, **<Open source plugin name>** // description here.
 
 For information on the **<Open source plugin name>** plugin see xref:<plugincode>.adoc[<Open source plugin name>].
 
+[[new-non-open-source-plugin]]
+== New Non Open Source plugin
```

Comment:
@lostkeys Are you happy with docs referring to these specific plugins as `non-open-source-plugins`?